src/applications/vaos/project-cheetah/components/SecondDosePage.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as actions from '../redux/actions';
+import FormButtons from '../../components/FormButtons';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import moment from 'moment';
+import { getReviewPage } from '../redux/selectors';
+import AdditionalInfoRow from '../../appointment-list/components/cards/AdditionalInfoRow';
+
+const pageKey = 'secondDosePage';
+const pageTitle = 'Plan your second dose';
+
+function goBack({ routeToPreviousAppointmentPage, history }) {
+  return routeToPreviousAppointmentPage(history, pageKey);
+}
+
+function SecondDosePage({
+  data,
+  facility,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const [showMoreOpen, setShowMoreOpen] = useState(false);
+  const history = useHistory();
+  const { date1, vaFacility } = data;
+
+  useEffect(() => {
+    if (history && !vaFacility) {
+      history.replace('/new-project-cheetah-booking');
+    }
+    document.title = `${pageTitle} | Veterans Affairs`;
+    scrollAndFocus();
+  }, []);
+
+  return (
+    <div>
+      <h1>{pageTitle}</h1>
+      <div>
+        <p>
+          You'll need to return to the {facility.name} for your second dose.
```

Comment:
I think this apostrophe should be the quote character, `’`

Reply:
done